### DISCUSSION TITLE
Why is the connector offset

### DISCUSSION THREAD

**Discussion by flycodec** [bcakmakoglu/vue-flow#723] [Q&A]
I set the id type label and position, and write the source, target and id connection lines in the edge to the left and the upper layer of the connection   。Only two setNodes and setEdges are used for re-rendering

**Comment by flycodec** [comment]
![image](https://user-images.githubusercontent.com/93771295/221751598-23f348b7-917c-4f1e-ad55-1dc8605d4803.png)
like this

**Comment by flycodec** [comment]
I set the id type label and location, and put the source, target and id connectors on the left edge of the upper layer, and used setNodes and setEdges for re-rendering

**Comment by flycodec** [comment]
However, if I use addNodes and addEdges, there will be no offset. Do I need to change my mind to add after each reset

**Comment by bcakmakoglu** [comment]
I'm not sure what you mean by offset? Do you have an image to compare it to?

**Reply by flycodec** [reply]
![image](https://user-images.githubusercontent.com/93771295/222057550-484d4a25-63d7-446d-8d17-b13ab74892ba.png)
Now this is the normal connection. The line has no offset

**Reply by flycodec** [reply]
let nozzleType = fn.node.label
    let { data } = await findEquipmentList({ aircraftNumber, nozzleType })
    let nodes = []
    let edges = []
    data.forEach((i, index) => {
      nodes.push({ id: i.modelNumber + index, label: i.equipmentType, position: { x: ++index * 50, y: 200 } })
    })
    nodes.forEach((i, index) => {
      if (index !== i.length - 1) {
        edges.push({ id: 'ell' + index, source: `${fn.node.id}`, target: `${i.id}` })
      }
    })
    topoEchartsList.value.forEach((i, index) => {
      nodes.unshift({ id: index, label: i.nozzleType, data: i.modelNumber, position: { x: ++index * 50, y: 30 } })
    })
    setNodes(nodes)
    setEdges(edges)

**Reply by bcakmakoglu** [reply]
Do you have a codesandbox for this?
Any warnings/errors you receive?
What version are you using?

**Reply by flycodec** [reply]
version ：1.15.1
no warnings/errors
The above is the event code I added
sorry，i don‘t have a codesandbox.

**Reply by bcakmakoglu** [reply]
Can you try downgrading to 1.14.3 or another version and check if it makes a difference? Helps narrowing down the issue possibly 😄

**Reply by flycodec** [reply]
ok
